projects/hatom-booster/index.js

```diff
@@ -0,0 +1,44 @@
+const { call } = require('../helper/chain/elrond')
+const { cachedGraphQuery } = require('../helper/cache')
+const { nullAddress } = require('../helper/tokenMapping')
+
+const boosterContractAddress = 'erd1qqqqqqqqqqqqqpgqw4dsh8j9xafw45uwr2f6a48ajvcqey8s78sstvn7xd'
+
+async function getMoneyMarkets() {
```

Comment:
I think this function is not needed, you can pass HTM token id directly like I did in my commit

Reply:
Yeap, you're right. Changed ✔️